cmd/syncer/options/options.go

```diff
@@ -50,11 +52,13 @@ func NewOptions() *Options {
 
 func (options *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&options.FromKubeconfig, "from-kubeconfig", options.FromKubeconfig, "Kubeconfig file for -from cluster.")
+	fs.StringVar(&options.FromContext, "from-context", options.FromContext, "Context to use in the Kubeconfig file for -from cluster, instead of the current context.")
+	fs.StringVar(&options.FromServer, "from-server", options.FromServer, "The overridden address and port of the Kubernetes API server to use in the Kubeconfig file for -from cluster.")
```

Comment:
already commented earlier. Why this change? Seems unrelated and unneeded.

Reply:
still open

Reply:
done